packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -73,8 +73,8 @@ const HomeLatestPosts = ({classes}:{classes: ClassesType}) => {
   const { captureEvent } = useOnMountTracking({eventType:"frontpageFilterSettings", eventProps: {filterSettings, filterSettingsVisible: filterSettingsVisibleDesktop, pageSectionContext: "latestPosts"}, captureOnMount: true})
   const { query } = location;
   const {
-    SingleColumnSection, PostsList2, TagFilterSettings, LWTooltip, SettingsButton, Typography,
-    CuratedPostsList, CommentsListCondensed, SectionTitle
+    SingleColumnSection, PostsList2, TagFilterSettings, LWTooltip, SettingsButton,
+    CuratedPostsList, SectionTitle, StickiedPosts,
   } = Components
   const limit = parseInt(query.limit) || 13
```

Comment:
The list feels a bit too long now the individual items are bigger 🤔, maybe reduce it by one or two?

Reply:
Done - reduced to 11